src/makei/cvtsrcpf.py

```diff
@@ -66,10 +67,12 @@ def insert_line(self, file_path, content, start_comment_characters: str, end_com
             return False
         
     def import_member_text(self, file_path: str, member_text: str, member_extension: str) -> bool:
-        text_comment_exists = self.check_keyword_in_file(file_path, 'TEXT', 15)
-        
-        # Text comment already exists
-        if text_comment_exists: return False
+        # Check if member text exists
+        metadata_comment_exists = self.check_keyword_in_file(file_path, '%METADATA', 15)
+        if metadata_comment_exists:
+            text_comment_exists = self.check_keyword_in_file(file_path, '%TEXT', 15)
```

Comment:
if you would take the line the %METADATA is found on and start searching from there to 15, that would be more efficient